transport/errors.go

```diff
@@ -13,7 +33,8 @@ func (e *ErrNotFound) Error() string {
 }
 
 func (e *ErrNotFound) Is(target error) bool {
-	_, ok := target.(*ErrNotFound)
+	var errNotFound *ErrNotFound
+	ok := errors.As(target, &errNotFound)
 	return ok
```

Comment:
```suggestion
	return errors.As(target, &ErrNotFound{})
```

Reply:
Your change seems like the right thing todo here, but the linter complains about it. 

```
transport/errors.go:38:9: errorsas: second argument to errors.As must be a non-nil pointer to either a type that implements error, or to any interface type (govet)
```

I will remove the `ok` as that isn't needed.